### ISSUE TITLE
Hasn't been updated in a while

### ISSUE THREAD

**Issue by rchl** [sublimelsp/LSP-cspell#7]
Would it make sense to sync with https://github.com/streetsidesoftware/vscode-spell-checker ?

**Comment by predragnikolic** [comment]
Yes,

Also, another truth is that there is no need for a language server like this to be updated frequently...
I been using for a couple of mounts
and it works fine in the current state as it is. 
- It underlines misspelled words
- It suggests code actions to fix those mistakes

TODO:
- implement `cSpell.fixSpellingWithRenameProvider` is a nice to have, but didn't invest my time to implement that.

I will try to update the server, 
IIRC, i run into some issues for some newer versions of the server, but maybe I will have more success this time.

**Comment by rchl** [comment]
There is a bunch of functionality that would could implement on the LSP-cspell side I've looked a bit at the code of vscode's extension and it feels like one of the most complex servers I've ever seen and its code is for some reason kinda hard to follow.

I guess it works so it's not that important.

**Comment by rchl** [comment]
by a "bunch of functionality" I guess I mainly mean adding ignore words in workspace specific file.
